jobs/nr-garbage-collector/nr_garbage_collector.py

```diff
@@ -59,6 +60,17 @@ def run_nr_garbage_collection():
     cancelled_nrs = []
 
     try:
+        # keeping nro separately
+        nro_reqs = db.session.query(Request).join(Payment). \
+            filter(
+                Request._source == ValidSources.NRO.value,
+                Request.stateCd != State.CANCELLED,
+                Payment._payment_status_code == PaymentStatusCode.CREATED.value,
+                Request.lastUpdate <= text(f"(now() at time zone 'utc') - INTERVAL '{delay} SECONDS'")). \
+            order_by(Request.lastUpdate.asc()). \
+            limit(max_rows). \
+            with_for_update().all()
+
         reqs = db.session.query(Request). \
             filter(or_(Request.stateCd.in_((State.COND_RESERVE, State.RESERVED)),
                        and_(Request.stateCd == State.DRAFT, or_(Request._source == 'NAMEREQUEST',
```

Comment:
since you're importing ValidSources can you add the NAMEREQUEST value to the comparison here?

Reply:
Will this pick NRS for which an upgrade or extension request is also pending? Why is Pending Payment state not considered here